spec/rollbar/plugins/active_job_spec.rb

```diff
@@ -9,6 +9,11 @@ class TestJob
     include Rollbar::ActiveJob
 
     attr_reader :job_id
+    attr_accessor :arguments
+
+    def initialize(*arguments)
```

Comment:
Actually this is confusing, ActiveJob kinda does it this way, `peform_now` basically calls `perform(*arguments)` where arguments is set in init. So to get these to match, the arguments should match up with what is passed to perform because all of this is fake.